src/analyze/BuildSystem.re

```diff
@@ -219,7 +229,7 @@ let getStdlib = (base, buildSystem) => {
     switch (Utils.getEnvVar(~env, "OCAMLLIB")) {
     | Some(esy_ocamllib) => Ok([esy_ocamllib])
     | None =>
-      let%try_wrap esy_ocamllib = getLine("esy -q sh -- -c 'echo $OCAMLLIB'", ~pwd=base);
+      let%try_wrap esy_ocamllib = getLine("esy b echo $OCAMLLIB", ~pwd=base);
```

Comment:
TODO: need to fix this - the previous command doesn't work on Windows w/ our current calling strategy (but works from the shell!). I used the `esy b` variant for testing which works, but has the undesirable property that it triggers a build... I'd like to see if we could use something along the previous lines.

Reply:
I agree that triggering a build is not perfect...

Reply:
Fixed this by tweaking the original command - the single quotes don't work the same way in Windows cmd shell, so we need double quotes (otherwise, they are treated as two separate arguments - `'echo` and `$OCAMLLIB'`. Used `Filename.quote` in [0dfbb35](https://github.com/jaredly/reason-language-server/pull/199/commits/0dfbb3569371af17c601d2ce3f87d644926142e4) to address this.